firestore/app/src/main/java/com/google/example/firestore/kotlin/DocSnippets.kt

```diff
@@ -940,7 +940,7 @@ abstract class DocSnippets(val db: FirebaseFirestore) {
     fun offlineListen(db: FirebaseFirestore) {
         // [START offline_listen]
         db.collection("cities").whereEqualTo("state", "CA")
-                .addSnapshotListener(EventListener<QuerySnapshot> { querySnapshot, e ->
+                .addSnapshotListener(MetadataChanges.INCLUDE, EventListener<QuerySnapshot> { querySnapshot, e ->
```

Comment:
This line is 112 chars long.  Is that okay? Or is there a formatter we use on these files?

Reply:
LGTM.  We don't auto-format snippet files.